cpp/src/arrow/csv/writer_test.cc

```diff
@@ -233,7 +241,15 @@ std::vector<WriterTestParams> GenerateTestCases() {
       reject_structural_params({"0123456789", nullptr, "abcde,", nullptr}, "abcde,"),
       reject_structural_params({"0123456789", nullptr, "abcdef,", nullptr}, "abcdef,"),
       reject_structural_params({nullptr, nullptr, ",0123456789", "abcde"}, ",0123456789"),
-      reject_structural_params({"0123456", nullptr, "7\\\"89", ",abcdef"}, "7\"89")};
+      reject_structural_params({"0123456", nullptr, "7\\\"89", ",abcdef"}, "7\"89"),
+      {schema_custom_delimiter, batch_custom_delimiter,
```

Comment:
Please insert a comment, e.g., `// exercise custom delimiter`, to separate from simd tests above.

Reply:
Made the change.